README.md

```diff
@@ -84,9 +84,10 @@ In your local dev environment, create a file: `.env.local`
 
 ### Environment Variables used in project
 
-| Variable  | value              | Description                                 |
-| --------- | ------------------ | ------------------------------------------- |
-| TIME_ZONE | "America/Edmonton" | For consistent timezones in the application |
+| Variable            | value                       | Description                                 |
+| ------------------- | --------------------------- | ------------------------------------------- |
+| TIME_ZONE           | "America/Edmonton"          | For consistent timezones in the application |
+| EDIT_HACKATHON_CODE | [any string value you want] | The code inputted for editting/deleting     |
```

Comment:
Instead of this, the code should just be a random short word to enter on the UI. It isn't for security, since you must be an admin to call this, but rather to prevent mis-clicks. Similar to the AWS `type "permanently delete" to delete` prompts.